### PR TITLE
Chess AI

### PR DESCRIPTION
**Pull request by Eugene44-hub**

### CODE REVIEW

File: src/screens/games/components/Piece.tsx [hunk 1]
```diff
@@ -57,6 +57,13 @@ const Piece = ({
   const offsetY = useSharedValue(0);
   const translateX = useSharedValue(startPosition.x * SIZE);
   const translateY = useSharedValue(startPosition.y * SIZE);
+  const AIMoves = () => {
```
Comment (jbrainz) [inline src/screens/games/components/Piece.tsx:60]: is this the AI logic?

Comment (Eugene44-hub) [reply src/screens/games/components/Piece.tsx:60]: > is this the AI logic?

Yes it is

Comment (jbrainz) [reply src/screens/games/components/Piece.tsx:60]: Kinda looks very basic,

**Review by jbrainz — APPROVED** [review]

File: src/screens/games/components/Piece.tsx (hunk 1 above)
Comment (Eugene44-hub) [reply src/screens/games/components/Piece.tsx:65]: That's the only way i could do it for now

**Pull request merged**